x-pack/test/detection_engine_api_integration/utils/prebuilt_rules/get_installed_rules.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import type SuperTest from 'supertest';
+import { DETECTION_ENGINE_RULES_URL_FIND } from '@kbn/security-solution-plugin/common/constants';
+import { FindRulesResponse } from '@kbn/security-solution-plugin/common/api/detection_engine';
+
+/**
+ * Get all installed security rules (both prebuilt + custom)
+ *
+ * @param es Elasticsearch client
+ * @param supertest SuperTest instance
+ * @param version Semver version of the `security_detection_engine` package to install
+ * @returns Fleet install package response
+ */
+
+export const getInstalledRules = async (
```

Comment:
nit: IMHO `fetchInstalledRules` would be the best name for the functionality under the hood as it clearly describes that it fetches the data presumable from an endpoint. While `get` usually used for sync functions extracting some data like `Map.get`. At the same time `getInstalledRules` conforms with the other functions in the folder like `getPrebuiltRulesStatus`. Ideally `getPrebuiltRulesStatus` should be renamed to `fetchPrebuiltRulesStatus` as well as the others but it definitely out of scope of this PR.

Reply:
Makes sense. I'll address this in any of the following PRs I'm working on. Thanks for all the feedback!